pkg/githubsource/receive_adapter.go

```diff
@@ -30,6 +30,12 @@ import (
 	gh "gopkg.in/go-playground/webhooks.v3/github"
 )
 
+const (
+	GHHeaderEvent     = "X-GitHub-Event"
+	GHHeaderDelivery  = "X-GitHub-Delivery"
+	CEExtensionPrefix = "CE-"
```

Comment:
how about reusing this ? 
https://github.com/knative/pkg/blob/master/cloudevents/encoding_binary.go#L60

Reply:
I didn't use it as it's private, but if you think it makes more sense I can push a PR over there.